packages/wonder-blocks-modal/components/modal-launcher-portal.js

```diff
@@ -31,17 +31,8 @@ export default class ModalLauncherPortal extends React.Component<Props> {
         destination.setAttribute("data-modal-launcher-portal", "");
         root.appendChild(destination);
 
-        // We'll also wrap the children in a wrapper node, so that we can
-        // accept `React.Node` instead of `React.Element`.
-        //
-        // NOTE(mdr): Once we upgrade to React 16, we'll no longer need this
-        //     wrapper node!
-        const wrapper = (
-            <View data-modal-launcher-portal>{this.props.children}</View>
-        );
-
         // Render the modal into the destination node.
-        ReactDOM.render(wrapper, destination);
```

Comment:
Was the wrapper not really necessary in the first place?

Reply:
It was necessary when `modal` was defined as accepting `React.Node`, because `ReactDOM.render` expects a `React.Element` instead of a `React.Node`.

Now that `modal` accepts a `React.Element`, we no longer need a wrapper element in order to be able to pass a single element here; we can just pass the `modal` itself.